x/ccv/provider/keeper/grpc_query.go

```diff
@@ -105,39 +107,34 @@ func (k Keeper) GetConsumerChain(ctx sdk.Context, consumerId string) (types.Chai
 	}, nil
 }
 
-// TODO (PERMISSIONLESS)
 func (k Keeper) QueryConsumerChainStarts(goCtx context.Context, req *types.QueryConsumerChainStartProposalsRequest) (*types.QueryConsumerChainStartProposalsResponse, error) {
+	return nil, status.Error(codes.Unimplemented, "This query is not supported anymore. Use `QueryConsumersThatAreAboutToStart` instead")
+}
+
+func (k Keeper) QueryConsumersThatAreAboutToStart(goCtx context.Context, req *types.QueryConsumersThatAreAboutToStartRequest) (*types.QueryConsumersThatAreAboutToStartResponse, error) {
```

Comment:
So a better name for this query is `QueryConsumersReadyToLaunch`, right? But the `GetInitializedConsumersReadyToLaunch` method returns consumers that are ready to launch now. This means that they will be launched in the next block. I'm not sure I understand the reason we have this query.

Reply:
Removed those queries altogether.